tests/wpt/web-platform-tests/xhr/setrequestheader-content-type.htm

```diff
@@ -215,23 +215,11 @@
         // https://fetch.spec.whatwg.org/#bodyinit, so the user's must be changed to match it
         // as per https://xhr.spec.whatwg.org/#the-send%28%29-method step 4.
       )
-      request(
-        function _ReadableStream() { return new ReadableStream() },
-        {"Content-Type": ""},
-        "",
-        'ReadableStream request respects setRequestHeader("")'
-      )
       request(
         function _ReadableStream() { return new ReadableStream() },
         {},
-        undefined,
-        "ReadableStream request with under type sends no Content-Type without setRequestHeader() call"
-      )
-      request(
-        function _ReadableStream() { return new ReadableStream() },
-        {"Content-Type": "application/xml;charset=ASCII"},
-        "application/xml;charset=ASCII",
-        "ReadableStream request keeps setRequestHeader() Content-Type and charset"
+        "text/plain;charset=UTF-8",
+        'ReadableStream request is treated as a String request."'
```

Comment:
This test doesn't verify anything about what actual bytes are present in the request due to how the readablestream object is converted to a string. I think it would be wise to do that to avoid accidental incompatibilities.

Reply:
Ok, thanks for checking, I think I have now added the right test, please take another look.